src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -977,7 +977,8 @@ public void StartInfo_NotepadWithContent(bool useShellExecute)
                 }
                 finally
                 {
-                    process.Kill();
+                    if (process != null && !process.HasExited)
+                        process.Kill();
```

Comment:
If process is null, we're also going to get null refs in the body of the try, e.g. on process.WaitForInputIdle, process.ProcessName, etc.